python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -261,7 +308,10 @@ def decorator_register(func):
     return decorator_register
 
 
-def generate_udf_resolvers(schema_name: str = "") -> List[ResolverSpec]:
+def generate_udf_resolvers(
+    schema_name: Union[str, List[str]] = "",
+    include_default_schema: bool = True,
```

Comment:
consider making this keyword only arg after the schema_name, e.g.
```
def generate_udf_resolvers(
    schema_name: : Union[str, List[str]] = "",
    *,
    include_default_schema: bool = True,
```

Reply:
Maybe if it grows more arguments? I'm ok with it as-is for now...